aries_vcx/src/protocols/proof_presentation/verifier/states/presentation_request_set.rs

```diff
@@ -1,13 +1,29 @@
+use messages::msg_fields::protocols::present_proof::request::{
+    RequestPresentation, RequestPresentationContent, RequestPresentationDecorators,
+};
+use uuid::Uuid;
+
 use crate::protocols::proof_presentation::verifier::states::presentation_request_sent::PresentationRequestSentState;
-use messages::protocols::proof_presentation::presentation_request::PresentationRequest;
 
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Default)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 pub struct PresentationRequestSetState {
-    pub presentation_request: PresentationRequest,
+    pub presentation_request: RequestPresentation,
+}
+
+impl Default for PresentationRequestSetState {
```

Comment:
compiles without `Default`

Reply:
Same as the other comment, I'd say it's outside of the PR.